SignalMessaging/ViewControllers/OWSNavigationController.m

```diff
@@ -0,0 +1,146 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSNavigationController.h"
+#import <SignalMessaging/SignalMessaging-Swift.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UINavigationController (OWSNavigationController) <UINavigationBarDelegate, NavBarLayoutDelegate>
+
+@end
+
+#pragma mark -
+
+@interface OWSNavigationController () <UIGestureRecognizerDelegate>
```

Comment:
Why two separate categories?  Why not just:

```
+@interface OWSNavigationController () <UIGestureRecognizerDelegate, UINavigationBarDelegate, NavBarLayoutDelegate>
```

Reply:
Eh, I see why you did this.

Reply:
I added a comment - it wasn't obvious.